CalendarPicker/index.js

```diff
@@ -39,6 +39,10 @@ export default class CalendarPicker extends Component {
       ...this.createMonths(props, {}),
     };
     this.state.renderMonthParams = this.createMonthProps(this.state);
+    Text.defaultProps = {
+      ...(Text.defaultProps || {}),
```

Comment:
This creates an unnecessary object (`{}`) because the spread operator does a no-op if `defaultProps` is null or undefined. Performance and readability can be improved by simply doing:
```
...Text.defaultProps,
```

Reply:
On further review, the defaultProps is replacing itself which creates a new object too. This should be rewritten as
```
 Text.defaultProps.allowFontScaling = props.fontScaling;
```